### PR TITLE
Task 2.5 complete

### PR DESCRIPTION
**Pull request by Deryck41**

### CODE REVIEW

**Comment by Deryck41** [comment]
[Задания №2](https://trello.com/c/wXm0OpQK/101-%D0%B7%D0%B0%D0%B4%D0%B0%D0%BD%D0%B8%D1%8F-%E2%84%962)

File: Task2-5/Task2-5/Program.cs [hunk 1]
```diff
@@ -0,0 +1,54 @@
+﻿using System;
```
Comment (PaShkadun) [inline Task2-5/Task2-5/Program.cs:1]: The same.

File: Task2-5/Task2-5/Program.cs [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+            Console.Write("Side 2: ");
+            bool secondSuccessInput = float.TryParse(Console.ReadLine(), out float secondSide);
+
+            Console.Write("Side 3: ");
+            bool thirdSuccessInput = float.TryParse(Console.ReadLine(), out float thirdSide);
+
+            if (firstSuccessInput && secondSuccessInput && thirdSuccessInput)
+            {
+                if (firstSide + secondSide > thirdSide && firstSide + thirdSide > secondSide && secondSide +
+                    thirdSide > firstSide)
```
Comment (PaShkadun) [inline Task2-5/Task2-5/Program.cs]: Don't use it here.

File: Task2-5/Task2-5/Program.cs [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+                {
+                    Console.WriteLine("\nThis triangle exists");
+                    if (firstSide == secondSide && secondSide == thirdSide)
```
Comment (PaShkadun) [inline Task2-5/Task2-5/Program.cs]: Whiteline.

File: Task2-5/Task2-5/Program.cs (hunk 2 above)
Comment (Deryck41) [reply Task2-5/Task2-5/Program.cs]: Do I need to solve this task in a different way or do I have problems with line break?

File: Task2-5/Task2-5/Program.cs [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+            Console.Write("Side 2: ");
+            bool secondSuccessInput = float.TryParse(Console.ReadLine(), out float secondSide);
+
+            Console.Write("Side 3: ");
+            bool thirdSuccessInput = float.TryParse(Console.ReadLine(), out float thirdSide);
+
+            if (firstSuccessInput && secondSuccessInput && thirdSuccessInput)
+            {
+                if (firstSide + secondSide > thirdSide && firstSide + thirdSide > secondSide && secondSide + thirdSide > firstSide)
```
Comment (PaShkadun) [inline Task2-5/Task2-5/Program.cs]: more or equal.

File: Task2-5/Task2-5/Program.cs [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+                {
+                    Console.WriteLine("\nThis triangle exists");
+
+                    if (firstSide == secondSide && secondSide == thirdSide)
+                    {
+                        Console.WriteLine("This triangle is equilateral");
+                    }
+
```
Comment (PaShkadun) [inline Task2-5/Task2-5/Program.cs]: No whiteline.

**Pull request merged**